js/main.js

```diff
@@ -115,12 +131,70 @@ $anchorTag.addEventListener('click', function () {
 }
 );
 
+// button for new entry form
 $anchorTagTwo.addEventListener('click', function () {
+  $deleteButton.classList.add('hidden');
+  $saveDeleteButtons.classList.add('single-button');
   viewSwap('entry-form');
 }
 );
 
-// this function to stay on same page during refresh as wall as swap views without reloading page
+// modal open function
+$deleteButton.addEventListener('click', function (event) {
+  $modal.classList.remove('hidden');
+  $body.classList.add('overlay');
+  $notes.classList.add('dark');
+  $title.classList.add('dark');
+  $photo.classList.add('opacity');
+  $img.classList.add('dark');
+  $mainHeader.classList.add('opacity');
+  $deleteButton.classList.add('opacity');
+  $saveButton.classList.add('opacity');
+  $formHeader.classList.add('opacity');
```

Comment:
There shouldn't be a need to give so many different elements a class of `opacity` or dark to apply the filter on them. You should just have a background color on your `modal` container, and that will impact everything